client/app/components/OnboardingTopBar/OnboardingTopBar.scss

```diff
@@ -0,0 +1,84 @@
+.topbarContainer {
+  width: 100%;
+  max-width: 120em;
+  color: white;
+  padding-top: 1.2em;
+  padding-bottom: 1.2em;
+  background-image: -moz-linear-gradient( 180deg, rgb(89,179,162) 0%, rgb(89,165,179) 100%);
+  background-image: -webkit-linear-gradient( 180deg, rgb(89,179,162) 0%, rgb(89,165,179) 100%);
+  background-image: -ms-linear-gradient( 180deg, rgb(89,179,162) 0%, rgb(89,165,179) 100%);
+  box-shadow: 0px 5px 8px 0px rgba(0, 0, 0, 0.06);
+}
+
+.topbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 63em;
+  margin: 0 auto;
+  padding-left: 0.75em;
+  padding-right: 0.75em;
+  position: relative;
+}
+
+.progressLabel {
+  display: inline-block;
+  font-weight: 600;
+  font-family: sans-serif;
+  font-size: 16px;
+  flex-shrink: 0;
+  margin-right: 0.5em;
+}
+
+.progressLabelPercentage {
+  font-size: 21px;
+  margin-left: 0.3em;
+}
+
+.progressBar {
+  background-color: rgb(184, 234, 225);
+  box-shadow: 0px 5px 8px 0px rgba(0, 0, 0, 0.06);
+  height: 100%;
+  border-radius: 0.5em;
+  transition: width 2s;
+}
+
+.progressBarBackground {
+  @media (max-width: 600px) {
```

Comment:
My line of thinking was to keep this as plain CSS as possible, so we can easily move to whichever postcss setup we decide. But keeping things in vars probably does make it easier, instead of more difficult.